webcompat/form.py

```diff
@@ -235,7 +236,7 @@ def build_formdata(form_object):
 **URL**: {url}
 **Problem type**: {problem_type}
 **Description**: {description}
-**Steps to Reproduce** {steps_reproduce}
+**Steps to Reproduce**: {steps_reproduce}
```

Comment:
Can you put `{steps_reproduce}` on the next line? That will take care of the "new line" bit, thanks.